controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -20,60 +20,46 @@
     /** Deus ex machina. */
     hostedOperator(Policy.operator),
 
+    /** Reader - the base role for all tenant users */
+    reader(
+            Policy.tenantRead,
+            Policy.applicationRead,
+            Policy.deploymentRead,
+            Policy.publicRead
+    ),
+
+    /** User - the dev.ops. role for normal Vespa tenant users */
+    developer(
+            Policy.applicationCreate,
+            Policy.applicationUpdate,
+            Policy.applicationDelete,
+            Policy.applicationOperations,
+            Policy.developmentDeployment
+    ),
+
+    /** Admin - the administrative function for user management etc. */
+    administrator(
+            Policy.tenantUpdate,
```

Comment:
This should be moved to developer. It currently covers nothing, and will cover developer key management.

Reply:
Hmm... We would probably want to ability to change tenant billing settings at some point, though. Perhaps we rather need two separate paths.

Reply:
Ok, moved things around:

Deploy keys are now POST/DELETE at `application/v4/tenant/{tenant}/application/{application}/key`, and developer keys at `application/v4/tenant/{tenant}/key`.

There's a `Policy.keyManagement` to allow these actions. Please insert this under the `developer` role above.

Reply:
(When it's available.)

Reply:
Or, I rather put the key stuff under tenantOperator for now, just for testing. It was simpler.